src/mltcontroller.cpp

```diff
@@ -934,10 +935,14 @@ Producer* Controller::setupNewProducer(Producer* newProducer) const
                 }
                 filter.reset(newProducer->filter(i));
             }
-            return chain;
+            ret = chain;
```

Comment:
This changed caused a severe regression:
1. Open a clip from File > Open or Recent
2. Add a filter
3. Upon choosing a filter, Shotcut hangs

Hmm, I just experienced strong deja vu!

Reply:
fixed in fe3f8355